test/helpers.js

```diff
@@ -153,19 +153,20 @@ Runner.prototype.npmVersion = function(version){
 	if(arguments.length === 0) {
 		return this._version; 
 	}
-
-	this.algorithm = version >= 3 ? "flat": undefined;
 	this._version = version;
-	this._addVersion();
```

Comment:
Yeah, I sort of agree, should set the algorithm with package.system in `rootPackage`, the helper sort of obscures what is going on, so I'm ok with getting rid of it.